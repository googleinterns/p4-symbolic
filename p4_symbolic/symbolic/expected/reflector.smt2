; 
(set-info :status unknown)
(declare-fun standard_metadata.ingress_port () (_ BitVec 9))
(assert
 (let (($x46 (= standard_metadata.ingress_port (_ bv1 9))))
 (or (or false (= standard_metadata.ingress_port (_ bv0 9))) $x46)))
(assert
 (and (and (not (or false false)) true) (= (- 1) (- 1))))
(check-sat)

