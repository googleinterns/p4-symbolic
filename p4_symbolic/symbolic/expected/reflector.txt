Finding packet for table tbl_reflector54 and row -1
	standard_metadata.ingress_port = #b000000000
	standard_metadata.egress_spec = #b000000000

